appservice/src/connectToGitHub.ts

```diff
@@ -63,9 +63,11 @@ export async function connectToGitHub(node: AzureTreeItem, client: SiteClient, c
         isMercurial: false
     };
 
+    const repoName: string | undefined = orgQuickPick.login && repoQuickPick.name ? `${orgQuickPick.login}/${repoQuickPick.name}` : undefined;
+
     try {
-        const connectingToGithub: string = localize('ConnectingToGithub', '"{0}" is being connected to the GitHub repo. This may take several minutes...', client.fullName);
-        const connectedToGithub: string = localize('ConnectedToGithub', '"{0}" has been connected to the GitHub repo.', client.fullName);
+        const connectingToGithub: string = repoName ? localize('ConnectingToGithub', '"{0}" is being connected to repo "{1}". This may take several minutes...', client.fullName, repoName) : localize('ConnectingToGithub', '"{0}" is being connected to the GitHub repo. This may take several minutes...', client.fullName);
```

Comment:
Do we really need two separate messages here? Seems unlikely that repoName would be undefined. You're not checking for undefined when you show it in the quick pick are you?

Reply:
Yeah, I agree it seems low-risk, was just being thorough.  I am checking for undefined in the quick pick though: https://github.com/Microsoft/vscode-azuretools/blob/1fd5e3b0b1e36f90f514831a2d872776f4e3e8d3/appservice/src/connectToGitHub.ts#L148.

If there's no label property, it just skips the entry entirely.

Reply:
... which means that it's guaranteed to have that data then